README.md

```diff
@@ -43,6 +43,59 @@ Now:
 * visit `/hello` in your browser
 
 
+## Reverse routing
+Exprestive exports reverse routing functions to a `paths` object. By default it is saved to `res.locals.paths` for easy access from views.
```

Comment:
+1  reverse routing is what the Play Framework calls it so I just decided to use that. I like "url building functions"